hist/hist/src/HFitImpl.cxx

```diff
@@ -894,14 +907,14 @@ TFitResultPtr ROOT::Fit::UnBinFit(ROOT::Fit::UnBinData * data, TF1 * fitfunc, Fo
 
       assert(  (Int_t)fitResult.Parameters().size() >= fitfunc->GetNpar() );
       fitfunc->SetParameters( const_cast<double*>(&(fitResult.Parameters().front())));
-      if ( int( fitResult.Errors().size()) >= fitfunc->GetNpar() ) 
-         fitfunc->SetParErrors( &(fitResult.Errors().front()) ); 
-  
+      if ( int( fitResult.Errors().size()) >= fitfunc->GetNpar() )
```

Comment:
This is a very odd way to convert `size_t` to int. Can't we change `GenNpar` to return `size_t` type?